init.lua

```diff
@@ -20,20 +25,19 @@ dofile(areas.modpath.."/hud.lua")
 areas:load()
 
 minetest.register_privilege("areas", {
-	description = "Can administer areas."
+	description = S("Can administer areas.")
 })
 minetest.register_privilege("areas_high_limit", {
-	description = "Can can more, bigger areas."
+	description = S("Can can more, bigger areas.")
 })
 
 if not minetest.registered_privileges[areas.config.self_protection_privilege] then
 	minetest.register_privilege(areas.config.self_protection_privilege, {
-		description = "Can protect areas.",
+		description = S("Can protect areas."),
 	})
 end
 
 if minetest.setting_getbool("log_mod") then
```

Comment:
Please use the new settings method.
```suggestion
if minetest.settings:get_bool("log_mods") then
```